macros/contexts/contextCurrency.pl

```diff
@@ -250,13 +250,9 @@ sub new {
 			precedence    => 10,
 			associativity => $associativity,
 			type          => "unary",
-			string        => (
-				($main::displayMode eq 'TeX' or $main::displayMode eq 'PTX')
-				? Currency::quoteTeX($symbol)
-				: $symbol
-			),
-			TeX   => Currency::quoteTeX($symbol),
-			class => 'Currency::UOP::currency'
+			string        => (($main::displayMode eq 'TeX') ? Currency::quoteTeX($symbol) : $symbol),
```

Comment:
Could you remove the extra parentheses on this line?  Now that this is all on one line those are excessive.  The only parentheses that are needed are those around `$symbol`.